samples/net/gptp/boards/mimxrt1020_evk.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_PTP_CLOCK_MCUX=y
```

Comment:
Has this been tested on `RT1020`

Reply:
I did some tests back in the day. gPTP is not working as expected in any imxrt model, including the imxrt1020, imxrt1050 and imxrt1060. According to the chip documentation, the imxrt1020 does have IEEE1588 hardware support. This line only enables the feature in the gptp sample application for this model.